src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -9,10 +9,9 @@
     <ProjectGuid>{89F37791-6254-4D60-AB96-ACD3CCA0E771}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);FEATURE_CORECLR</DefineConstants>
```

Comment:
/cc @davidsh 
Thanks @weshaggard . I think i'll wait for this before I complete #11489.